python/lsst/ts/MTMount/mock/controller.py

```diff
@@ -137,79 +113,68 @@ def __init__(
         self.command_dict[enums.CommandCode.SAFETY_RESET] = self.do_safety_reset
 
         self.read_loop_task = asyncio.Future()
-        self.telemetry_task = asyncio.Future()
+        self.telemetry_loop_task = asyncio.Future()
+        self.telemetry_monitor_task = asyncio.Future()
```

Comment:
Good question. I do this in two different files: in one it is because the file doesn't need salobj for anything else and I'm reluctant to add a dependency just for that. In the other I don't have much of a reason beyond "make it look like the other file".

I really wish we had a lightweight repository for simple things like this.

Reply:
I asked because this particular file does import salobj. And perhaps you (or we) should collect use cases like this one and propose a lightweight ts_common project to put all that into?